src/gateways/createIntl.ts

```diff
@@ -0,0 +1,18 @@
+import { createIntl, createIntlCache } from 'react-intl';
+import enMessages from '../app/messages/en';
+import plMessages from '../app/messages/pl';
+
+export default () => {
+  return {
+    getIntlObject: (locale: string = 'en') => {
```

Comment:
how much would it suck to make this async? it'd be nice to do a dynamic `import` rather than loading all the messages for every language

Reply:
so I have this working but the catch is now a whole lot of tests have to be made async (which is maybe what you were alluding to)

Reply:
yea i can see how a lot of tests would be broken... i'm struggling to think of a way to make this better without that problem.

i definitely don't want to be loading every language file and then picking one, and since we don't know the language until the book loads i'm not sure if there is any way around it